test/exercises_test.dart

```diff
@@ -126,5 +128,5 @@ void main() {
 
       await runTest(testPath);
     }
-  }, timeout: new Timeout(new Duration(seconds: 120)));
+  }, timeout: new Timeout(new Duration(minutes: 10)));
```

Comment:
5 Minutes should suffice?

Reply:
The more exercises we add to the longer the tests will take. So it's going to get progressively longer. I've had to extend it to 3 and half minutes. So I'd rather punt it

The point of the timeout is to stop the process in case there's an infinite loop or some problem (hardware or other software) that could case the tests to go on for too long.